modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -0,0 +1,44 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
+
+[id='modifying-install-config-for-dual-stack-network_{context}']
+
+= Modifying the `install-config.yaml` file for dual-stack network (optional)
```

Comment:
I wonder about writing these as:

> Optional: Modifying...

to conform with ISG?

Reply:
There are numerous modules with that configuration. So I think we'll wait to post release and fix them all, so that it doesn't stick out like a sore thumb.